docs/examples.rst

```diff
@@ -1180,7 +1180,7 @@ The script can be run with: ``python ./eventscanner.py <your JSON-RPC API URL>``
         """
 
         if from_block is None:
-            raise TypeError("Missing mandatory keyword argument to getLogs: fromBlock")
+            raise TypeError("Missing mandatory keyword argument to get_logs: from_block")
```

Comment:
Not sure if the code example here needs additional updates based on the changes in the PR here. I think it does, but would someone be able to take a look and double check me? @fselmo @pacrob @kclowes 

A few lines below in this file, `event_filter_params` is passed into `get_logs` like so:  
```
    data_filter_set, event_filter_params = construct_event_filter_params(
        abi,
        codec,
        address=argument_filters.get("address"),
        argument_filters=argument_filters,
        fromBlock=from_block,
        toBlock=to_block
    )

    logger.debug(f"Querying eth_getLogs with the following parameters: {event_filter_params}")

    # Call JSON-RPC API on your Ethereum node.
    # get_logs() returns raw AttributedDict entries
    logs = w3.eth.get_logs(event_filter_params)
```

Reply:
It looks like the `construct_event_filter_params` function def still has it's args camelCased.  But you could update there and then the docs too.

Reply:
Okay - sounds good. I'll leave this where it's at then, request another review, and have a separate PR to update `construct_event_filter_params` in the docs. 

Thanks for double checking me!